jekyll/_cci2/github-trigger-event-options.adoc

```diff
@@ -14,21 +14,37 @@ When you create a trigger for a GitHub App pipeline, you can choose from a set o
+CircleCI projects have pipelines and trigger.
 
-.Trigger options during project creation
-image::triggers/create-project-run-on.png[Create a project trigger and access trigger options]
+* Pipelines define the various CI/CD 'units' of work that make up your project.
+* Triggers define when and how to initiate a pipeline.
+
+Configure triggers under menu:Project settings[Triggers]. You can configure triggers to initiate pipelines on a variety of events
 
-Once your project is set up you can create additional pipelines and triggers (menu:Project settings[Triggers]). You can configure your triggers to initiate pipelines on a variety of events, including non-push events, for example, a pull request being opened, or refined push events, for example, only pushes to a default branch, or pushes to a tag. For a full list see <<supported-trigger-options>>.
+* Non-push events, for example, a pull request being opened
+* Push events, for example, only pushes to a default branch, or pushes to a tag.
+
+For a full list see <<supported-trigger-options>>.
 
 .Trigger options under Project settings
 image::triggers/run-on-open.png[Set a trigger option for a project trigger]
 
+When you create or set up a project in CircleCI you will set up your first pipeline and trigger. This first trigger is, by default, an "all push" trigger, which corresponds to the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads#push[GitHub App push event].
+
+include::../_includes/partials/tips/check-github-type-org.adoc[]
+
+If you have a **GitHub App** account (see tip above) with CircleCI, you will see trigger options when creating a project in the CircleCI web app, and you have the option to change the trigger options for your first trigger. You can also set up additional triggers during project creation, or setup/edit later in your menu:Project settings[Triggers].
+
+.Trigger options during project creation
+image::triggers/create-project-run-on.png[Create a project trigger and access trigger options]
+
+If you have a **GitHub OAuth** integration (see tip above) with CircleCI, an org admin can install the CircleCI GitHub App to enable GitHub app pipeline functionality, including the trigger options described on this page. Installing the CircleCI GitHub App is a one-time action. For more information, see the xref:github-integration#installing-github-app-alongside-github-oauth-app[GitHub OAuth integration overview].
+
 == Supported trigger options
 
 [NOTE]
 ====
-Regardless of your chosen trigger option, CircleCI will never trigger a pipeline for the following:
+Regardless of your chosen trigger option, CircleCI will **never** trigger a pipeline for the following:
 
 * On branch or tag deletion push events.
 * When a new branch is created if there are no new commits on the branch.
```

Comment:
Ugh I've just realised - this last point is true for GitHub App but not true for OAuth

Reply:
```suggestion
GitHub App pipelines are also never trigger on the following:
* When a new branch is created if there are no new commits on the branch.
```